waspc/examples/todoApp/ext/pages/auth/Login.js

```diff
@@ -1,16 +1,24 @@
 import React, { useState } from 'react'
 import { Link } from 'react-router-dom'
+import { useQuery } from '@wasp/queries'
 
 import LoginForm from '@wasp/auth/forms/Login'
+import { getNumTasks } from '@wasp/queries/getNumTasks.js'
 
 const Login = (props) => {
+  const { data: tasks, isError, error: tasksError } = useQuery(getNumTasks)
+  // console.log(useQuery(getNumTasks()))
   return (
     <>
       <LoginForm/>
       <br/>
       <span>
         I don't have an account yet (<Link to="/signup">go to signup</Link>).
       </span>
+      <br/>
+      <span>
+        Number of tasks already created: 
+      </span>
```

Comment:
We could also put this on signup form? I would put it on both sign up and login form! And if we have to choose between them, then better just signup form, since that is where "new users" will come first and we want to impress them with how many tasks have already been created with Todo App :D!

Reply:
Done, moved to the Signup page